From ead26c4d7dbdd72c9566fe800743f6fcff5f0f82 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sat, 14 Mar 2020 01:28:28 +0100
Subject: [PATCH] madplay: disable Native Language support for now

With enabled NLS, the compilations fails with:
/bin/sh: 0: Can't open @MKINSTALLDIRS@
---
 sound/madplay/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/madplay/Makefile b/sound/madplay/Makefile
index 574bda959..ab9025958 100644
--- a/sound/madplay/Makefile
+++ b/sound/madplay/Makefile
@@ -55,6 +55,7 @@ CONFIGURE_ARGS += \
 	--without-libiconv-prefix \
 	--without-libintl-prefix \
 	--without-esd \
+	--disable-nls \
 
 CONFIGURE_VARS += \
 	lt_prog_compiler_pic=$(FPIC)
-- 
2.25.1

